SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -33,62 +35,77 @@
-        <script src="lib/angular-uuids/angular-uuid.js"></script>
+    </nav>
+    <div class="container" ng-view></div>
+    <script src="lib/jquery/dist/jquery.js"></script>
+    <script src="lib/jquery-ui/jquery-ui.js"></script>
+    <script src="lib/lodash/dist/lodash.js"></script>
+    <script src="lib/angular/angular.js"></script>
+    <script src="lib/angular-resource/angular-resource.js"></script>
+    <script src="lib/angular-route/angular-route.js"></script>
+    <script src="lib/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+    <script src="lib/angular-ui-date/src/date.js"></script>
+    <script src="lib/jquery.nicescroll/jquery.nicescroll.js"></script>
+    <script src="lib/sweetalert2/dist/sweetalert2.js"></script>
+    <script src="lib/select2/dist/js/select2.js"></script>
+    <script src="lib/moment/moment.js"></script>
+    <script src="lib/bootstrap-css/js/bootstrap.min.js"></script>
+    <script src="lib/fingerprint/fingerprint.js"></script>
+    <script src="lib/angular-web-crypto/dist/ng-web-crypto.es5.js"></script>
+    <script src="lib/angular-uuids/angular-uuid.js"></script>
 
 
-        <script src="js/app.js"></script>
-        <script src="js/home/home-controller.js"></script>
-        <script src="js/login/login-controller.js"></script>
-        <script src="js/register/register-controller.js"></script>
-        <script src="js/register/register-service.js"></script>
-        <script src="js/portal/portal-controller.js"></script>
+    <script src="js/app.js"></script>
+    <script src="js/home/home-controller.js"></script>
+    <script src="js/login/login-controller.js"></script>
+    <script src="js/register/register-controller.js"></script>
+    <script src="js/register/register-service.js"></script>
+    <script src="js/portal/portal-controller.js"></script>
+    <script src="js/profiles/user-profile-controller.js"></script>
 
+    <script src="js/chat/chat-controller.js"></script>
+    <script src="js/chat/chat-service.js"></script>
+    <script src="directives/chat-message/chat-message.js"></script>
+    <script src="directives/chat-list/chat-list.js"></script>
+    <script src="directives/input-field-validation/unique-field-validation.js"></script>
+    <script src="directives/input-field-validation/password-confirmation.js"></script>
 
-        <script src="js/chat/chat-controller.js"></script>
-        <script src="js/chat/chat-service.js"></script>
+    <script src="js/chat/chat-controller.js"></script>
+    <script src="js/chat/chat-service.js"></script>
 
-        <script src="directives/chat-message/chat-message.js"></script>
-        <script src="directives/chat-list/chat-list.js"></script>
-        <script src="directives/input-field-validation/unique-field-validation.js"></script>
-        <script src="directives/input-field-validation/password-confirmation.js"></script>
+    <script src="directives/chat-message/chat-message.js"></script>
```

Comment:
The below 4 javascript files are duplicates. Please kindly remove them. The chat page is broken because of this. In addition, please verify that nothing else has changed. I might be missing something.